async def pass_error(*_):
    return True
